<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>714</width>
    <height>640</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
    background-color: #f4f4f4;
    font-size: 14px;
    color: #333;
}

QPushButton {
    background-color: #007AFF;
    color: white;
    border-radius: 10px;
    padding: 8px 16px;
    font-weight: 500;
	transition: all 0.3s ease;
}

QPushButton:hover {
    background-color: #006FCC;
	box-shadow: 0 0 12px rgba(25, 118, 210, 0.6);
}

QPushButton:pressed {
    background-color: #003E99;
}

QLineEdit, QSpinBox, QDoubleSpinBox {
    border: 1px solid #ccc;
    border-radius: 8px;
    padding: 8px;
    background-color: white;
    font-size: 15px;
}

/* SpinBox: ซ่อนปุ่มลูกศรให้ดูคลีน */
QSpinBox::up-button, QSpinBox::down-button,
QDoubleSpinBox::up-button, QDoubleSpinBox::down-button {
    width: 0px;
    height: 0px;
    border: none;
}

/* GroupBox */
QGroupBox {
    border: 1px solid #e0e0e0;
    border-radius: 12px;
    margin-top: 10px;
    padding: 10px;
}

QGroupBox::title {
    subcontrol-origin: margin;
    left: 10px;
    padding: 0 4px;
    font-weight: bold;
}

/* QListWidget */
QListWidget {
    border: 1px solid #ccc;
    border-radius: 8px;
    padding: 5px;
    background-color: #fafafa;
}

QLabel {
    font-size: 14px;
    color: #333;
}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>80</y>
      <width>401</width>
      <height>431</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Prompt</family>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="locale">
     <locale language="English" country="UnitedStates"/>
    </property>
    <property name="title">
     <string>รายการอื่นๆ</string>
    </property>
    <widget class="QLineEdit" name="otherNameEdit">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Prompt</family>
       <pointsize>-1</pointsize>
      </font>
     </property>
     <property name="locale">
      <locale language="English" country="UnitedStates"/>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="otherAmountSpin">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>30</y>
       <width>71</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Prompt</family>
       <pointsize>-1</pointsize>
      </font>
     </property>
     <property name="locale">
      <locale language="English" country="UnitedStates"/>
     </property>
     <property name="decimals">
      <number>0</number>
     </property>
     <property name="minimum">
      <double>-999999.000000000000000</double>
     </property>
     <property name="maximum">
      <double>999999.000000000000000</double>
     </property>
    </widget>
    <widget class="QPushButton" name="addOtherButton">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>30</y>
       <width>61</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Prompt</family>
       <pointsize>-1</pointsize>
       <fontweight>Medium</fontweight>
      </font>
     </property>
     <property name="locale">
      <locale language="English" country="UnitedStates"/>
     </property>
     <property name="text">
      <string>เพิ่ม</string>
     </property>
    </widget>
    <widget class="QListWidget" name="otherListWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>361</width>
       <height>321</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Prompt</family>
       <pointsize>-1</pointsize>
      </font>
     </property>
     <property name="locale">
      <locale language="English" country="UnitedStates"/>
     </property>
    </widget>
    <widget class="QPushButton" name="removeOtherButton">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>30</y>
       <width>61</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Prompt</family>
       <pointsize>-1</pointsize>
       <fontweight>Medium</fontweight>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">#removeOtherButton {
    background-color: #ff524b;
    color: white;
    border-radius: 10px;
    padding: 8px 16px;
    font-weight: 500;
    transition: all 0.2s;
}

#removeOtherButton:hover {
    background-color: #FF3B30;
}</string>
     </property>
     <property name="locale">
      <locale language="English" country="UnitedStates"/>
     </property>
     <property name="text">
      <string>ลบ</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="remainingLabel">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>550</y>
      <width>271</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Prompt</family>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="locale">
     <locale language="English" country="UnitedStates"/>
    </property>
    <property name="text">
     <string>เงินที่เหลือ</string>
    </property>
   </widget>
   <widget class="QPushButton" name="saveButton">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>550</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Prompt</family>
      <pointsize>-1</pointsize>
      <fontweight>Medium</fontweight>
     </font>
    </property>
    <property name="locale">
     <locale language="English" country="UnitedStates"/>
    </property>
    <property name="text">
     <string>บันทึก</string>
    </property>
   </widget>
   <widget class="QLabel" name="totalSaveLabel">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>570</y>
      <width>261</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Prompt</family>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="locale">
     <locale language="English" country="UnitedStates"/>
    </property>
    <property name="text">
     <string>เงินเก็บทั้งหมด</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>190</y>
      <width>221</width>
      <height>211</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Prompt</family>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="locale">
     <locale language="English" country="UnitedStates"/>
    </property>
    <property name="title">
     <string>ค่าอาหาร</string>
    </property>
    <widget class="QSpinBox" name="breakfastSpin">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>42</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Prompt</family>
       <pointsize>-1</pointsize>
      </font>
     </property>
     <property name="locale">
      <locale language="English" country="UnitedStates"/>
     </property>
     <property name="maximum">
      <number>999</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>40</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Prompt</family>
       <pointsize>-1</pointsize>
      </font>
     </property>
     <property name="locale">
      <locale language="English" country="UnitedStates"/>
     </property>
     <property name="text">
      <string>อาหารเช้า</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="lunchSpin">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>42</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Prompt</family>
       <pointsize>-1</pointsize>
      </font>
     </property>
     <property name="locale">
      <locale language="English" country="UnitedStates"/>
     </property>
     <property name="maximum">
      <number>999</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>100</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Prompt</family>
       <pointsize>-1</pointsize>
      </font>
     </property>
     <property name="locale">
      <locale language="English" country="UnitedStates"/>
     </property>
     <property name="text">
      <string>อาหารกลางวัน</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="dinnerSpin">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>42</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Prompt</family>
       <pointsize>-1</pointsize>
      </font>
     </property>
     <property name="locale">
      <locale language="English" country="UnitedStates"/>
     </property>
     <property name="maximum">
      <number>999</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>160</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Prompt</family>
       <pointsize>-1</pointsize>
      </font>
     </property>
     <property name="locale">
      <locale language="English" country="UnitedStates"/>
     </property>
     <property name="text">
      <string>อาหารเย็น</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="Head">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>10</y>
      <width>721</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Prompt</family>
      <pointsize>-1</pointsize>
      <strikeout>false</strikeout>
      <kerning>true</kerning>
      <fontweight>Medium</fontweight>
     </font>
    </property>
    <property name="locale">
     <locale language="English" country="UnitedStates"/>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:26pt;&quot;&gt;จัดการการเงินประจำวัน&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>221</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Prompt</family>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="locale">
     <locale language="English" country="UnitedStates"/>
    </property>
    <property name="title">
     <string>รายรับวันนี้</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>25</y>
       <width>151</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Prompt</family>
       <pointsize>-1</pointsize>
      </font>
     </property>
     <property name="locale">
      <locale language="English" country="UnitedStates"/>
     </property>
     <property name="text">
      <string>ใส่จำนวนเงินที่ได้ในวันนี้</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="incomeEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>151</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Prompt</family>
       <pointsize>-1</pointsize>
      </font>
     </property>
     <property name="locale">
      <locale language="English" country="UnitedStates"/>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="viewHistoryButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>550</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Prompt</family>
      <pointsize>-1</pointsize>
      <fontweight>Medium</fontweight>
     </font>
    </property>
    <property name="locale">
     <locale language="English" country="UnitedStates"/>
    </property>
    <property name="text">
     <string>ประวัติการใช้เงิน</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>410</y>
      <width>221</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Prompt</family>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="locale">
     <locale language="English" country="UnitedStates"/>
    </property>
    <property name="title">
     <string>เงินเก็บวันนี้</string>
    </property>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>25</y>
       <width>171</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Prompt</family>
       <pointsize>-1</pointsize>
      </font>
     </property>
     <property name="locale">
      <locale language="English" country="UnitedStates"/>
     </property>
     <property name="text">
      <string>ใส่จำนวนเงินที่จะเก็บในวันนี้</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="saveBankEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>151</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Prompt</family>
       <pointsize>-1</pointsize>
      </font>
     </property>
     <property name="locale">
      <locale language="English" country="UnitedStates"/>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="totalSaveLabel_2">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>590</y>
      <width>261</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Prompt</family>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="locale">
     <locale language="English" country="UnitedStates"/>
    </property>
    <property name="text">
     <string>เงินเเก็บเข้าธนาคาร</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>714</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
